src/Main/App.js

```diff
@@ -285,6 +286,49 @@ class App extends Component {
       .then(response => response.json());
   }
 
+  renderContent() {
```

Comment:
Please put this method above the `render()` method so that everything can be found easily.